AssemblyUtil.spec

```diff
@@ -25,6 +25,24 @@ module AssemblyUtil {
     funcdef get_assembly_as_fasta(GetAssemblyParams params) 
                 returns (FastaAssemblyFile file) authentication required;
 
+    /*
+           Structure for get_fasta function input and output:
+            rKBaseOjbReferences - is an array of KBase object references, such as KBaseGenomes.Genome, KBaseSets.AssemblySet, etc.
+            FASTA - is an array of fasta file paths
+    */
+
+    typedef list<string> KBaseOjbReferences;
+
+     typedef structure {
+        list<string> FASTA;
+    } GetFASTAOutput;
+
+    /*
+        Given a reference list of KBase objects constructs a local Fasta file with the sequence data for each ref.
+
+    */
+    funcdef get_fastas(KBaseOjbReferences ref_lst)
```

Comment:
Here's where I would actually wrap the list in a structure. The reason for this is that you can add keys to structures and have the KIDL API be backwards compatible. You can't do that for list<string>. 

You *don't* need to do this for internal APIs since they're abstracted away from the user calling the KIDL API.